apps/api/src/hyp3_api/validation.py

```diff
@@ -66,7 +66,7 @@ def get_cmr_metadata(granules):
 
 
 def is_third_party_granule(granule):
-    return granule.startswith('S2') or granule.startswith('L')
+    return granule.startswith('S2') or granule.startswith('L') or granule.endswith('-BURST')
```

Comment:
I think this might deserve a #FIXME, or some other way to call out that it's temporary until we integrate hyp3-isce2 with production CMR

Reply:
Added a `FIXME` and an issue: https://github.com/ASFHyP3/hyp3/issues/1651